webapp/app.py

```diff
@@ -151,13 +151,17 @@ def global_template_context():
     version_parts = VANILLA_VERSION.split(".")
     version_minor = f"{version_parts[0]}.{version_parts[1]}"
 
-    docs_slug = (
-        flask.request.path.replace("/docs/", "")
-        .replace("/design/", "")
-        .replace("/accessibility", "")
-    )
+    # Add an exception for the /docs/search path
+    if flask.request.path == "/docs/search":
+        docs_slug = ""
+    else:
+        docs_slug = (
+            flask.request.path.replace("/docs/", "")
+            .replace("/design/", "")
+            .replace("/accessibility", "")
+        )
 
-    docs_slug = "" if docs_slug == "/docs" else docs_slug
+        docs_slug = "" if docs_slug == "/docs" else docs_slug
```

Comment:
Why not add the exception here, where we already have a `"/docs` exception

Not sure if that would work but something like:
```
docs_slug = "" if docs_slug == "/docs" or docs_slug == "/docs/search" else docs_slug
```

Reply:
That was the first thing that I tried, but actually the `/docs/` part of the path gets already replaced one instruction before:
```
        docs_slug = (
            flask.request.path.replace("/docs/", "")
            .replace("/design/", "")
            .replace("/accessibility", "")
        )
```
so this condition: `docs_slug == "/docs/search"` will never be matched.

Reply:
Ah, right!